src/electron/main.ts

```diff
@@ -12,10 +13,12 @@ function createWindow() {
   const window = new BrowserWindow({
     width: 1400,
     height: 800,
+    title: 'ESA – Climate from Space',
     webPreferences: {
       nodeIntegration: false,
       contextIsolation: true,
-      preload: path.join(__dirname, 'preload.js')
+      preload: path.join(__dirname, 'preload.js'),
+      enableRemoteModule: true
```

Comment:
why do you enable the remote module? it is deprecated and should not be used. see https://www.electronjs.org/docs/breaking-changes#default-changed-enableremotemodule-defaults-to-false

Reply:
Oh, I thought we need it and wanted to get rid of the warning. Why else would it warn me, if I didn't use it? Will remove it.

Reply:
Yeah I was confused here too at first, but they added the warning because they changed the default value for the property which could break a lot of apps.

Reply:
Keeping it, since electron 10 default is set to false and its not trivial to move away from it.

Reply:
Added a new ticket #527  to remove the remote module